package.json

```diff
@@ -6,7 +6,8 @@
   "repository": "mozilla/normandy-actions",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "./node_modules/.bin/webpack --progress",
+    "build": "./node_modules/.bin/webpack --progress --colors",
+    "watch": "./node_modules/.bin/babel-node ./bin/watch.js",
```

Comment:
AFAIK most of the downsides an inherent with not compiling babel code beforehand, so we'd have to figure out a sane way of building these beforehand.